Utilities.Rest/Packages/com.utilities.rest/Runtime/Rest.cs

```diff
@@ -494,13 +494,14 @@ public static async Task<Response> DeleteAsync(
 
         private const string download_cache = nameof(download_cache);
 
-        private static IDownloadCache cache;
-
-        private static IDownloadCache Cache => cache ??= Application.platform switch
-        {
-            RuntimePlatform.WebGLPlayer => new NoOpDownloadCache(),
-            _ => new DiskDownloadCache()
-        };
+        /* We start with a thread-safe *stub* cache so that background threads can safely call Rest even before Unity’s main thread runs Init_Rest().
+         * This No-op cache never touches Unity APIs and stores nothing on disk.
+         * Until Init_Rest() swaps in DiskDownloadCache, helpers that rely on the
+         * cache (e.g. DownloadTextureAsync / DownloadAudioClipAsync / DownloadFileAsync) will simply redownload the asset each time.
+         */
+        private static IDownloadCache cache = new NoOpDownloadCache();
+        // Returns the already-initialized cache. Call Init_Rest() once on the main thread before using Rest from worker threads.
+        private static IDownloadCache Cache => cache;
```

Comment:
just make this a property w/o field backing
```suggestion
        private static IDownloadCache Cache { get; private set; }
```